docs/modules/ROOT/pages/capabilities-explained.adoc

```diff
@@ -30,19 +53,14 @@ This allows them to do that.
 ==== Analytics Support
 
 This is where the library has a mechanism to log an event, potentially attach metadata.
-The library captures the state of all of the features at the point in time of the request and will pass it on to any registered Analytics provider.
+The library captures the state of all the features at the point in time of the request and will pass it on to any registered Analytics provider.
 A platform can have analytics support but no analytics providers.
 We intend over time to support only one, where the data is posted to a backend service which you can then decide where to send and how to send the data.
 
 ==== Google Analytics
 
 This is a client side implementation of the Analytics support.
-It is designed so you need to specify the `User-ID` or `CID`, your `UA-` id and when logging an event, it will fire off into GA the event - one for each value of non-JSON features.
-
-=== Web + Mobile Capabilities
-
-The following capabilities are focused on clients that provide a UI to the client and thus you may wish to control
-the updating of the features.
+You need to specify the `User-ID` or `CID`, your `UA-` id and when logging an event, it will fire off into GA the event - one for each value of non-JSON features.
```

Comment:
This is UA rather than GA4. Might be  worth removing this until we seed GA4  through  everywhere.

Reply:
it is UA! i havent made any changes in it